src/components/OutputEditor.tsx

```diff
@@ -63,15 +63,17 @@ export default function OutputEditor({
   }, [monaco])
 
   const handleViewModeChange = (event: ChangeEvent<HTMLSelectElement>) => {
-    onViewModeChange(event.target.value)
+    startTransition(() => {
```

Comment:
This is really pedantic and unnecessary. Nobody will change view mode too many times.